docs/installing.rst

```diff
@@ -33,7 +33,11 @@ Install the latest development snapshot from the `GitHub repository's master bra
 
 1. Create a new conda environment called ``opencadd``::
 
-    mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/test_env.yaml -n opencadd
+    mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/user_env.yaml -n opencadd-dev
```

Comment:
```suggestion
    mamba env create -f https://raw.githubusercontent.com/volkamerlab/opencadd/master/devtools/conda-envs/user_env.yaml 
```
Since `user_env.yaml` is the file excluding packages for testing and documentation, we can skip the `-n` here. The new env will be assigned accordingly to what is specified in `user_env.yml`.